.eslintrc

```diff
@@ -11,5 +11,14 @@
   "rules": {
     "no-param-reassign": [2, { "props": false }],
     "func-names": ["error", "never"]
-  }
+  },
+  "overrides": [
+    {
+      "excludedFiles": ["app/assets/javascripts/arclight/context_navigation.js"],
```

Comment:
Following https://eslint.org/docs/user-guide/configuring#disabling-rules-only-for-a-group-of-files, I couldn't manage to get this passing without violations.  I will attempt adjusting this.

Reply:
I think thats fine for now.. Inline block disables could also work? (maybe there was an issue with that?)

Reply:
This doesn't seem to be effective (please see https://travis-ci.org/projectblacklight/arclight/jobs/596661383), I'll try inline block disables.